lib/spack/docs/build_systems/sippackage.rst

```diff
@@ -108,21 +115,25 @@ a "package" is a directory containing files like:
    foo/baz.py
 
 
-whereas a "module" is a single Python file. Since ``find_packages``
-only returns packages, you'll have to determine the correct module
-names yourself. You can now add these packages and modules to the
-package like so:
+whereas a "module" is a single Python file.
+
+The ``SIPPackage`` base class automatically detects these module
+names for you. If, for whatever reason, the module names detected
+are wrong, you can provide the names yourself by overriding
+``import_modules`` like so:
 
 .. code-block:: python
 
    import_modules = ['PyQt5']
 
 
-When you run ``spack install --test=root py-pyqt5``, Spack will attempt
-to import the ``PyQt5`` module after installation.
+These tests often catch missing dependencies and non-RPATHed
+libraries. Make sure not to add modules/packages containing the word
```

Comment:
As of now, `import_modules` doesn't ignore `test*` modules. When I originally wrote this in #15370, I was using `setuptools.find_packages`, which has a tool for this. It would probably be good to add something similar to the default `import_modules` implementation. Do you want that to happen in this PR or would a follow-up PR be fine?

Reply:
FWIW.  I think a follow-up PR is fine (or preferred).